### PR TITLE
Console review part2 fixes without role and permission changes

### PR DESCRIPTION
**Pull request by bojanutkovic**
Signed-off-by: Bojan Utkovic <bojan.utkovic@comtrade.com>

### CODE REVIEW

File: console/module/api/src/main/resources/org/eclipse/kapua/app/console/module/api/client/messages/ConsoleMessages.properties [hunk 1]
```diff
@@ -275,7 +278,7 @@ accountProvisionedConfirmation=Provisioning for Account {0} started.
 
 accountNoSelectedAccount=No Selected Account
 accountFormNew=New Account
-accountFormUpdate=Update Account: {0}
+accountFormUpdate=Edit Account: Name: {0}
```
Comment (lorthirk) [inline console/module/api/src/main/resources/org/eclipse/kapua/app/console/module/api/client/messages/ConsoleMessages.properties]: Are you sure about this string?

File: console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/client/group/GroupFilterPanel.java [hunk 2]
```diff
@@ -37,7 +39,7 @@ public GroupFilterPanel(AbstractEntityView<GwtGroup> entityView, GwtSession curr
         super(entityView, currentSession);
         entityGrid = entityView.getEntityGrid(entityView, currentSession);
         this.currentSession = currentSession;
-
+        setIcon(new KapuaIcon(IconSet.FILTER));
```
Comment (lorthirk) [inline console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/client/group/GroupFilterPanel.java]: We probably want to move this line in the base class constructor, so all the filters inherits it. Check also if other specialized filters sets it and remove it from there.

File: console/module/tag/src/main/java/org/eclipse/kapua/app/console/module/tag/client/TagFilterPanel.java [hunk 3]
```diff
@@ -37,7 +39,7 @@ public TagFilterPanel(AbstractEntityView<GwtTag> entityView, GwtSession currentS
         super(entityView, currentSession);
         entityGrid = entityView.getEntityGrid(entityView, currentSession);
         this.currentSession = currentSession;
-
+        setIcon(new KapuaIcon(IconSet.FILTER));
```
Comment (lorthirk) [inline console/module/tag/src/main/java/org/eclipse/kapua/app/console/module/tag/client/TagFilterPanel.java]: We probably want to move this line in the base class constructor, so all the filters inherits it. Check also if other specialized filters sets it and remove it from there.

File: console/module/api/src/main/resources/org/eclipse/kapua/app/console/module/api/client/messages/ConsoleMessages.properties [hunk 4]
```diff
@@ -278,7 +278,7 @@ accountProvisionedConfirmation=Provisioning for Account {0} started.
 
 accountNoSelectedAccount=No Selected Account
 accountFormNew=New Account
-accountFormUpdate=Edit Account: Name: {0}
+accountFormUpdate=Update Account: Name: {0}
```
Comment (lorthirk) [inline console/module/api/src/main/resources/org/eclipse/kapua/app/console/module/api/client/messages/ConsoleMessages.properties]: The correct string should be "Edit Account: {0}".

**Comment by LeoNerdoG** [comment]
@bojanutkovic @lorthirk I have reviewed this PR and it is OK. From my point of view it can be merged.

File: console/module/account/src/main/resources/org/eclipse/kapua/app/console/module/account/client/messages/ConsoleAccountMessages.properties [hunk 5]
```diff
@@ -25,7 +25,7 @@ enabled=Enabled
 
 #
 # Account Add dialog
-accountAddDialogHeader=New Account
+accountAddDialogHeader=New Child Account
```
Comment (lorthirk) [inline console/module/account/src/main/resources/org/eclipse/kapua/app/console/module/account/client/messages/ConsoleAccountMessages.properties]: This should be "New Account"

**Comment by muros-ct** [comment]
Tests are failing. Please run build without -DskipTests.
And rebase.

**Comment by bojanutkovic** [comment]
Hi @lorthirk, does it look ok now?

**Comment by lorthirk** [comment]
Looks good to me now. @LeoNerdoG, can you confirm it's ok to merge?

**Comment by LeoNerdoG** [comment]
@lorthirk i have reviewed this today and it is OK to merge.

**Pull request merged**